### PR TITLE
Title of plot configuration is not split any more

### PR DESCRIPTION
**Pull request by PavelBal**
Fixing issue described here https://github.com/esqLABS/esqlabsR.reports/issues/6

### CODE REVIEW

File: R/utilities-figures.R [hunk 1]
```diff
@@ -313,13 +313,18 @@ createPlotsFromExcel <- function(
...
-      c("plotID", "DataCombinedName", "plotType", "aggregation", "quantiles", "foldDistance"))],
-    1, .createConfigurationFromRow,
-    defaultConfiguration = plotConfiguration
-  )
+  defaultPlotConfiguration <- createEsqlabsPlotConfiguration()
+  plotConfigurationList <- apply(dfPlotConfigurations, 1, \(row){
+    plotConfiguration <- .createConfigurationFromRow(
+      defaultConfiguration = defaultPlotConfiguration,
+      # Have to exclude all columns that should not be vectorized
+      row[!(names(row) %in% c("plotID", "DataCombinedName", "plotType", "title", "aggregation", "quantiles", "foldDistance"))]
```
Comment (Felixmil) [inline R/utilities-figures.R:321]: shouldn't subtitle, caption, xLabel and yLabel be also excluded here ?

Comment (PavelBal) [reply R/utilities-figures.R:321]: You are right, this just never have been used so far.. Adding it.

File: R/utilities-figures.R [hunk 2]
```diff
@@ -313,13 +313,27 @@ createPlotsFromExcel <- function(
...
+    plotConfiguration <- .createConfigurationFromRow(
+      defaultConfiguration = defaultPlotConfiguration,
+      # Have to exclude all columns that should not be vectorized
+      row[!(names(row) %in% c("plotID",
+                              "DataCombinedName",
+                              "plotType",
+                              "title",
+                              "subtitle",
+                              "xLabel",
+                              "yLabel",
```
Comment (Felixmil) [inline R/utilities-figures.R:327]:
```suggestion
                              "ylabel",
```

**Comment by PavelBal** [comment]
@Felixmil no, the field in the `DefaultPlotConfiguration` is called `xLabel` (capital L), so the column name would be the same.

**Pull request merged**